Functions.Templates/ProjectTemplate_v4.x/FSharp-Isolated/Company.FunctionApp.fsproj

```diff
@@ -10,6 +10,8 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Functions.Worker" Version="1.19.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Worker.Sdk" Version="1.15.1" />
+    <PackageReference Include="Microsoft.ApplicationInsights.WorkerService" Version="2.21.0" />
```

Comment:
Should we also be include the package to C# template?

Reply:
Not sure what you mean by this (haven't worked on templates before) but same change is going into the C# template in #1474.

Reply:
Ahh i did not see that PR